cypresse2e/cypress/integration/e2eTests/privacyPolicy/privacyPolicy.spec.js

```diff
@@ -0,0 +1,168 @@
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the TENTH paragraph with Order List starting with-10. With Whom We Share Your Personal Data?", (dataType = "paraPolicyCxWithOrdList", ParaNum = 10) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+    it("Get and Validate the PRIVACY POLICY content for the ELEVENTH paragraph with Order List starting with-11. How long We store your Personal Data?", (dataType = "paraPolicyCxWithOrdList", ParaNum = 11) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the TWELVETH paragraph with Order List starting with-12. Where We Store Your Personal Data", (dataType = "paraPolicyCxWithOrdList", ParaNum = 12) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+    it("Get and Validate the PRIVACY POLICY content for the THIRTEENTH paragraph with Order List starting with-13. How We Protect Your Personal Data", (dataType = "paraPolicyCxWithOrdList", ParaNum = 13) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the FOURTEENTH paragraph with Order List starting with-14. Contact Us", (dataType = "paraPolicyCxWithOrdList", ParaNum = 14) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+
+    it("Get and Validate the PRIVACY POLICY content for the FIFTEENTH paragraph with Order List starting with-15. Changes to Our Privacy Policy", (dataType = "paraPolicyCxWithOrdList", ParaNum = 15) => {
+      const getMethod = privacyPolicy.getLocationParaWithOrdList(ParaNum);
+      validateEachParaWithOrdList(getMethod, dataType, ParaNum);
+    });
+    it("Get and Validate Date of Document created", () => {
+      privacyPolicy
+        .getDateDocumented()
+        .invoke("text")
+        .should("eq", "April 23rd, 2020");
+    });
+  });
+
+  //to validate each paragraph
+  function validateEachPara(dataType, ParaNum) {
+    const objData = getObjectData(dataType, ParaNum);
+    const text = objData.text;
+    var contentPara = privacyPolicy.getLocationPara(ParaNum).contains(text);
+  }
+
+  //to validate each paragraph with order list
+  function validateEachParaWithOrdList(getMethod, dataType, ParaNum) {
+    const objData = getObjectData(dataType, ParaNum);
+    const text = objData.text;
+    const email = objData.email;
```

Comment:
A suggestion is to create an email constant for "legal@fightpandemics.com" in this spec file and write a test that will find it throughout the context, instead of adding the email where it already exists in the json file. I'm not sure how often the privacy policy context changes, but an email can be removed or added to different paragraphs at any point. What do you think?

Reply:
@jessica-lau it makes sense thank you :)

Reply:
Done:). if you have a better way than my current , please let me know